doc/OPTIONS

```diff
@@ -437,14 +445,8 @@ especially for fast to compute hash types (such as LM hashes), where
 OpenMP overhead is often unacceptable.
 
 Similarly to "--node", there's almost no communication between the
-processes with "--fork".  Hashes successfully cracked by one process
-continue being cracked by other processes.  Just like with "--node",
-this is mostly OK for saltless hash types or when there's just one salt,
-but it is a serious drawback when many different salts are present and
-their number could potentially be decreasing as some hashes get cracked.
-To have the cracked hashes (and possibly salts) removed from all
-processes, you may interrupt and restore the session once in a while.
-
+processes with "--fork" but the "pot sync" feature (described under that
+option) will mitigate most of the problems.
```

Comment:
I'd add a comma before "but", and replace "mitigate most of the problems" with "promptly exclude hashes cracked by other processes".